app/src/androidTest/java/edu/nextstep/camp/calculator/MainActivityTest.kt

```diff
@@ -13,102 +13,66 @@ class MainActivityTest {
     val activityScenarioRule = ActivityScenarioRule(MainActivity::class.java)
 
     @Test
-    fun test0() {
-        // when: 사용자가 피연산자 0 버튼을 누르면
-        onView(withId(R.id.button0)).perform(click())
-
-        // then: 계산기 텍스트에 숫자 0이(가) 보여야 한다.
-        onView(withId(R.id.textView)).check(matches(withText("0")))
-    }
-
-    // 사용자가 피연산자 1 버튼을 누르면 화면에 숫자 1이(가) 보여야 한다.
-    @Test
-    fun test1() {
-        // when: 사용자가 피연산자 1 버튼을 누르면
+    fun click_1_2_3_shows_123() {
         onView(withId(R.id.button1)).perform(click())
-
-        // then: 계산기 텍스트에 숫자 1이(가) 보여야 한다.
-        onView(withId(R.id.textView)).check(matches(withText("1")))
+        onView(withId(R.id.button2)).perform(click())
+        onView(withId(R.id.button3)).perform(click())
+        onView(withId(R.id.textView)).check(matches(withText("123")))
     }
 
-    // 사용자가 피연산자 2 버튼을 누르면 화면에 숫자 2이(가) 보여야 한다.
     @Test
-    fun test2() {
-        // when: 사용자가 피연산자 2 버튼을 누르면
+    fun click_1_3_plus_2_2_shows_1_3_plus_2_2() {
```

Comment:
바로위에 작성해주신 테스트 코드 이름과 같이 보여지는 UI를 그대로 작성하는게 좋을 것 같습니다 
```suggestion
    fun click_1_3_plus_2_2_shows_13_plus_22() {
```